store/file/ods_q4.go

```diff
@@ -125,22 +121,15 @@ func (odsq4 *ODSQ4) Sample(ctx context.Context, rowIdx, colIdx int) (shwap.Sampl
 func (odsq4 *ODSQ4) AxisHalf(ctx context.Context, axisType rsmt2d.Axis, axisIdx int) (eds.AxisHalf, error) {
 	size := odsq4.Size(ctx) // TODO(@Wondertan): Should return error.
 
-	var acsr eds.Accessor = odsq4.ods
 	if axisIdx >= size/2 {
-		q4, err := odsq4.getQ4()
-		if err != nil {
-			return eds.AxisHalf{}, err
+		// lazy load Q4 file and read axis from it if loaded
+		odsq4.tryLoadQ4()
```

Comment:
Readability nit: i would still make it return q4, so it doesn't confuse readers with accessing Q4 field without synchronization, while it has it on the struct. Returning it makes it seem like the tryLoadq4 takes full responsibility on proper synchornization